src/js/main.js

```diff
@@ -400,21 +405,15 @@
             },
 
             render() {
-                cx.fillStyle = 'cyan';
-                // Different size for drawing than for collision checking.
-                let w = this.width * 1.2, h = this.height * 1.1;
-                let x = this.x - (w - this.width) * 0.5;
-                let y = this.y - (h - this.height) * 0.7 + 4;
+                cx.fillStyle = 'green';
+                let w = this.width * 1.2, h = this.height;
+                let x = this.x - (w - this.width);
```

Comment:
I think this should be `(w - this.width) * 0.5`. Half added there because the difference between the drawing width and the collision width should be distributed evenly on both the left side and the right side.

Now that the drawing height is the same as collision height, the y coordinate should work as it is now.